### PR TITLE
peridaxon works slower and doesn't mend bones

### PR DESCRIPTION
**Pull request by Firefox13**
## About The Pull Request
Fixes Peridaxon treating all wounds immediately (prob might be a little too low, but eh) and healing bones. It also doesn't heal brain now (use alkysine for this)

## Why It's Good For The Game
It's a fix.

## Testing
https://cdn.discordapp.com/attachments/265411250341543936/1109153257177239623/2023-05-19_19-11-37.mp4
Video that shows me breaking my ribcage and injecting peridaxon. I shoot myself several times afterwards to show that ribcage isn't healing but organs are.

## Changelog
:cl:
fix: Peridaxon doesn't heal brain or bones anymore. It's also slower at it.
fix: Peridaxon doesn't damage bones on OD too (brain is also spared).
/:cl:

### CODE REVIEW

**Comment by MLGTASTICa** [comment]
can you make it metabolize slower instead of adding a prob(25) it is litteraly hard to make and a top-tier chem.

File: code/modules/reagents/reagents/medicine.dm [hunk 1]
```diff
@@ -385,18 +385,19 @@
 /datum/reagent/medicine/peridaxon/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
-		for(var/obj/item/organ/I in H.internal_organs)
+		var/list/organs_sans_brain_and_bones = H.internal_organs - H.internal_organs_by_efficiency[BP_BRAIN] - H.internal_organs_by_efficiency[OP_BONE] // Peridaxon shouldn't heal brain or bones
+		for(var/obj/item/organ/I in organs_sans_brain_and_bones)
 			var/list/current_wounds = I.GetComponents(/datum/component/internal_wound)
-			if(LAZYLEN(current_wounds) && !BP_IS_ROBOTIC(I)) //Peridaxon heals only non-robotic organs
+			if(LAZYLEN(current_wounds) && !BP_IS_ROBOTIC(I) && prob(25)) //Peridaxon heals only non-robotic organs
```
Comment (Firefox13) [inline code/modules/reagents/reagents/medicine.dm:391]:
```suggestion
			if(LAZYLEN(current_wounds) && !BP_IS_ROBOTIC(I) && prob(75)) //Peridaxon heals only non-robotic organs
```
I'll do this.

Comment (Firefox13) [reply code/modules/reagents/reagents/medicine.dm:391]: Prob is there so it doesn't treat all wounds immediately.

Comment (MLGTASTICa) [reply code/modules/reagents/reagents/medicine.dm]: could also make it just pick one at random instead of fixing all.

**Review by hyperioo — APPROVED** [review]

**Pull request merged**